### DISCUSSION TITLE
No idea where to start

### DISCUSSION THREAD

**Discussion by davidnonu** [pawelsalawa/sqlitestudio#4936] [General]
### Details
I haven't touch our database since last June and as I add new databases right now, I get the "database disk image is malformed" error. I'm not a coding guy at all, I'm lucky I figured out how to even see the data from the sqlite files. Any help would be awesome as my new work depends on this to work well. I did check to see if the most recent version of SQLiteStudio is downloaded and it says it is.

### Steps to reproduce
Described above

### Operating system
Windows 11

### SQLiteStudio version
3.4.4

**Comment by pawelsalawa** [comment]
Maybe your database was encrypted with a password? In that case you need to add it to SQLiteStudio with appropriate enryption-supporting driver in the Database Dialog window and - obviously - correct password.

If encryption is not the issue, then I cannot help you. The "malformed" database means SQLite itself (not SQLiteStudio, but underlying SQLite driver) is not able to read this database.

**Reply by davidnonu** [reply]
Thank you for your info! I'll check out to see if its a password.
